.travis.yml

```diff
@@ -24,13 +24,14 @@ install:
     - source activate testenv
     - conda install --yes scipy numpy matplotlib
     - pip install flake8 pytest pytest-cov
-    - pip install mpi4py mne
+    - pip install mne psutil mpi4py joblib
```

Comment:
We might unwittingly introduce a dependency on these packages if we do it like this

There should be a CI that works with the minimal dependencies. See how we do it in [mne-bids](https://github.com/mne-tools/mne-bids/blob/master/.travis.yml#L21)

Reply:
That's kind of unfair to ask me to do this when you could have done it for joblib. This PR is big enough.

Reply:
That's kind of unfair to ask me to do this when you could have done it for joblib. This PR is big enough.

Reply:
@jasmainak Can you suggest how to do this in CircleCI?